analyzers/tests/SonarAnalyzer.Test/TestCases/SpecifyRouteAttribute.CSharp12.cs

```diff
@@ -190,3 +190,27 @@ public class DerivedController : Controller { }
 
 [Controller]
 public class Endpoint { }
+
+[Route("api/[controller]")]
+public class ControllerWithRouteAttribute : Controller { }
+
+public class ControllerWithInheritedRoute : ControllerWithRouteAttribute            // Compliant, attribute is inherited
+{
+    [HttpGet("Test")]
+    public string Index() => "Hi!";
+}
+
+public class BaseControllerWithActionWithRoute : Controller                         // Noncompliant
+{
+    [HttpGet("Test")]
```

Comment:
```suggestion
    [HttpGet("Test")] //Route: /Test (AmbiguousMatchException raised because of the override in ControllerOverridesActionWithRoute)
```